apps/services/handlers.py

```diff
@@ -270,5 +277,43 @@ async def __generate_user_document_name(cls, service_id: int, session: AsyncSess
         )
         return f"Заява на {service.service_name.lower()}"
 
+    async def create_students_from_file(
+            self,
+            *,
+            request: Request,
+            university_id: int,
+            file: UploadFile = File(),
+            session: AsyncSession):
+        check_content_type(file)
```

Comment:
maybe it's better to make method check_content_type as a decorator?

Reply:
We can make it as a decorator, of course. But what benefits will we gain from that? Decorator in that handlers file and throut the others will be the first one and will strike the eye...

Reply:
we need to do it bcs the validation of content type should be done before we run the method, not after... so it's just the right logic 1) validate data 2) run data